dashboard/app/assets/stylesheets/application.scss

```diff
@@ -4216,3 +4216,15 @@ a {
 .cm-editor {
   height: 100%;
 }
+
+[dir="rtl"] {
+  [data-markdown] {
+    [style*="float: left"] {
+      float: right !important;
```

Comment:
The better alternative would be using `"float: inline-start;"` but it is only supported by modern browsers:
https://caniuse.com/mdn-css_properties_float_inline-start

Reply:
You could write them both in, no? It will then fallback to the "less ideal" float rule?

Or do you think it is more correct to just use these more complicated rules and maybe just write a comment in to revisit when the browsers update on our IT page?

Reply:
Could you please clarify what you mean by "write them both in"? I believe we can only have one rule for the same property, as one would override the other

Reply:
Do you mean that one of the rules will be used as a fallback for the other if the browser version does not support it?

Reply:
Yes that. That is, I believe, how CSS is designed to work. It will use the proper rule when supported and use the other rule as a fallback when it does not understand the value (`inline-start` in this case).

Then, instead of writing a note to implement it a different way later... which we might forget... we can add a note to simply remove the fallback when we notice we can.